diff --git a/node_modules/react-native-ui-lib/react-native.config.js b/node_modules/react-native-ui-lib/react-native.config.js
index ef376f5..272982e 100644
--- a/node_modules/react-native-ui-lib/react-native.config.js
+++ b/node_modules/react-native-ui-lib/react-native.config.js
@@ -1,22 +1,8 @@
 module.exports = {
   dependency: {
+    // https://github.com/wix/react-native-ui-lib/issues/3805#issuecomment-3322406480
     platforms: {
-      /* TODO: Once we upgrade to RN69 we should try using podspecPath again, for now I copied ReactNativeUiLib.podspec file to the root - it seems to work
-      I copied it, because we need it for both the main uilib and the uilib-native package */
-      // ios: {
-      //   podspecPath: './lib/ReactNativeUiLib.podspec'
-      // },
-      android: {
-        sourceDir: '../uilib-native/android/',
-        packageImportPath: `import com.wix.reactnativeuilib.dynamicfont.DynamicFontPackage;
-import com.wix.reactnativeuilib.highlighterview.HighlighterViewPackage;
-import com.wix.reactnativeuilib.keyboardinput.KeyboardInputPackage;
-import com.wix.reactnativeuilib.textinput.TextInputDelKeyHandlerPackage;`,
-        packageInstance: `new DynamicFontPackage(),
-      new HighlighterViewPackage(),
-      new TextInputDelKeyHandlerPackage(),
-      new KeyboardInputPackage(getApplication())`
-      }
+      android: null, // Disable autolinking - native code comes from uilib-native dependency in package.json
     }
   }
 };
